opencsp/app/sofast/lib/ParamsOpticGeometry.py

```diff
@@ -8,10 +8,15 @@ class ParamsOpticGeometry(hdf5_tools.HDF5_IO_Abstract):
     """Parameter dataclass for processing optic geometry"""
 
     perimeter_refine_axial_search_dist: float = 50.0
+    """The length of the search box (along the search direction) to use when finding optic perimeter. Default 50.0"""
```

Comment:
Thank you for adding comments! Are these in units of pixels?

Reply:
Yes and done!